### DISCUSSION TITLE
Две новых анимации в F2

### DISCUSSION THREAD

**Discussion by Darling-in-Balbes** [octothorp-team/public-dobrograd#669] [Предложения]
Что добавить: предлагаю добавить две анимаций в F2 меню, а именно: лежать (ранение),  сидеть спиной (ранение). Эти анимации не так уж и сложно сделать самостоятельно, к тому же одна из анимаций есть в ХЛ2, из которой ее можно просто взять.

Зачем: поднять РП атмосферу. 

Пример: Кого-то ранили в живот, а он взял и упал на землю при помощи F2 анимации.Это будет выглядеть в разы лучше, чем если он тупо два раза нажмет CTRL и будет сидеть.

И это только самый очевидный пример. Уверен, что игроки придумают этим анимациям больше способов применения.

**Comment by Black-White228337** [comment]
https://forum-old.octothorp.team/d/4564-dobavit-animatsii-po-tipu-leg-na-zemlyu
Уже давно в планах
